demo/src/app/components/datepicker/demos/customday/datepicker-customday.ts

```diff
@@ -25,7 +25,7 @@ import {NgbDateStruct} from '@ng-bootstrap/ng-bootstrap';
     }
   `]
 })
-export class NgbdDatepickerCustomDay {
+export class NgbdDatepickerCustomday {
```

Comment:
I would probably prefer to rename files to `datepicker-custom-day.ts`
(but not important really)